data/json/recipes/recipe_deconstruction.json

```diff
@@ -506,6 +506,60 @@
     ]
   },
   {
+    "result": "makeshift_oscilloscope",
```

Comment:
I'm not sure I completely understand deconstruction. Where it documented or alternatively do you have a quick summary?